Frameworks/UIKit/UIActivityIndicatorView.mm

```diff
@@ -129,57 +133,49 @@ - (instancetype)initWithFrame:(CGRect)frame {
  Microsoft Extension
 */
 - (instancetype)initWithFrame:(CGRect)frame xamlElement:(RTObject*)xamlElement {
-    // TODO: We're passing nil to initWithFrame:xamlElement: because we have to *contain* a _subview for padding/layout.
-    // Note: Pass 'xamlElement' instead, once we move to a *single* backing Xaml element for UIActivityIndicatorView.
-    if (self = [super initWithFrame:frame xamlElement:nil]) {
-        [self _initUIActivityIndicatorView:xamlElement];
+    if (self = [super initWithFrame:frame xamlElement:xamlElement]) {
+        [self _initUIActivityIndicatorView];
     }
 
     return self;
 }
 
 /**
  @Status Interoperable
+ @Note Frame will not be resized smaller than the contained ProgressRing, as this would obscure it.
```

Comment:
Interesting; that sounds like the spinner is in a separate, non-clipped layer on iOS.  We could do something similar, but it seems like a corner case that we probably don't need to worry about.  However, I'm worried that silently capping the min frame size will break layout - cause layout cycles, etc.  The correct way to do this would be to implement intrinsicContentSize, but perhaps we just remove the capping and see how that works out for most apps (unless you already know of an app that requires this min frame size?)

---
In reply to: [113078524](https://github.com/Microsoft/WinObjC/pull/2608#discussion_r113078524) [](ancestors = 113078524)